openapi.yaml

```diff
@@ -21,6 +21,95 @@ tags:
   - name: comments
   - name: websocket
 components:
+  schemas:
+    authLoginResponse:
+      description: Authentication response
+      type: string
+    idResponse:
+      description: Response Containing an ID
+      type: object
+      properties:
+        id:
+          type: number
+          description: ID of newly created entity
+      required:
+        - id
+    comment:
+      description: A comment on a piece of content
+      type: object
+      properties:
+        id:
+          type: integer
+        text:
+          type: string
+        community_id:
+          type: integer
+        content_id:
+          type: integer
+        parent_id:
+          type: integer
+      required:
+        - id
+        - text
+        - community_id
+        - content_id
```

Comment:
Would it make more sense to nest the content type here and put the text within the content?

Reply:
I have no idea, maybe?